December 22, 2020

# Single-cell transcriptomic analysis of Alzheimer’s disease

[Full Text](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6865822/)
[Commentary](https://www.ncbi.nlm.nih.gov/pubmed/31216459)

This study examined 80,660 single nucleus transcriptomes from 48 brains, 24 of
which had elevated β-amyloid and other AD pathologies. 
Compared to bulk level analysis, snRNA-Seq was better able to capture the
complexity of alterations across cells and cell groups. 
They compared the gene expression levels between the AD-pathology and
no-pathology cells and found "1031 unique differentially-extressed genes (DEGs)
that implicate all major cell types". 

These genes showed repression in exitatory and inhibitory neurons and
upregulation in oligodendrocytes, astrocytes and microglia.  Compared to this,
the bulk analysis was only able to capture the expression from the single-cell
level in exitatory neurons and oligodendrocytes, not the other cell types. Bulk
sequencing could also not capture when the differentially-expressed genes had
opposite directionality.

The authors used an unsupervised learning framework called self-organizing maps
to find the gene sets with similar expression patterns that also correlated with
the AD phenotype. They found that in exitatory and inhibitory neurons,
astrocytes, microglia, and oligodendrocytes, each cell type had a distinct SOM
unit which meant that different gene groups respond to AD pathology.

Examining the pathology between sexes, they found differences in AD-associated
cell subpopulations, transcriptional activations, and responses to amyloid.
