---
date: Sept 9, 2020
---

# Single-Nucleus RNA-Seq Is Not Suitable for Detection of Microglial Activation Genes in Humans {.unnumbered}

## Summary

snRNA-seqs have failed to detect a consistent microglial activation signature in
AD. Using four subjects, they show that while most genes show relative abundance
in cells and nuclei, some genes are depleted in nuclei compared to whole cells.
These cells are enriched for genes implicated in microglial activation making up
18% of microglial-disease-associated genes.

## Intro

It is difficult to dissociate individual cells, especially in the brain since
most available samples are frozen. While snRNA-seq has been shown effective for
transcriptomic profiles, it is not clear if it works as effectively for dynamic
cellular substates like microglial activation.

Microglia have been shown to become activated in response to amyloid plaques.
Microglia play a role in amyloid clearance.

However snRNA-seq studies have not been able to recover a similar activation
signature from microglia.

## Results

Most genes show similar abundance levels between cells and nuclei. There were
however, 246 genes that were less abundant in nuclei and 67 genes were found to
be more abundant in nuclei. 3248 genes were only detected in cells and 5068 were
only found in nuclei. These differences were consistent across the four
subjects. Compared with other datasets, they found that their
nuclei-underrepresented genes were also depleted in the other datasets.
Similarly, the nuclei-abundant genes were also enriched in those datasets
compared to microglial cells.

A GSEA showed that cytoplasmic RNA was enriched among genes more abundant in
cells (also mitochondrial mRNA). Nucleus mRNA was also enriched in the nucleus
but not significantly. Ribosomal protein genes were depleted in the nucleus.
Genes with shorter coding sequences were depleted in the nuclei.

Activation genes identified in mouse AD models were depleted in human nuclei.

Checking with another study that used fresh cells for snRNA-seq analysis, they
found that there was very little difference between the fresh nuclei and the cells.
The differences returned when the frozen nuclei were compared to cells.

They were unable to attribute the differences to a biological mechanism like
mRNA trafficking. The differences between fresh and frozen nuclei suggests that
the freezing or thawing process could play a role.

They advise using caution when using snRNA seq to assess cellular states.
